Library/ViewModels/PledgeViewModel.swift

```diff
@@ -428,6 +431,82 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
       createBackingEvent.filter { $0.isTerminating }.mapConst(true),
       updateBackingEvent.filter { $0.isTerminating }.mapConst(true)
     )
+
+    // MARK: - Success/Failure Create
+
+    let createPaymentAuthorizationDidFinishSignal = Signal.zip(
```

Comment:
`zip` allows us to ensure that each of these signals will emit at least once before this signal emits. So we know that we started out creating here and not updating because we `zip` with the `willCreateApplePayBacking` signal.